### PR TITLE
feat: product-listing

### PR DESCRIPTION
**Pull request by KaustubhM1997**

### CODE REVIEW

File: pages/product-listing/product-listing.css [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+    font-size: 0.9rem;
+    padding-top: 1.1rem;
+}
+
+.rating-options label {
+    font-size: 0.9rem;
+}
+
+.item-conatiner {
```
Comment (srishti-maurya) [inline pages/product-listing/product-listing.css]: You may recheck for the spelling here

**Review by srishti-maurya — APPROVED** [review]
Looks good

File: pages/product-listing/product-listing.css (hunk 1 above)
Comment (KaustubhM1997) [reply pages/product-listing/product-listing.css]: Thank you for pointing it out.

**Review by manojkumar8400 — APPROVED** [review]
looks good to me.

**Pull request merged**